config/locales/en/summary.yml

```diff
@@ -634,114 +636,114 @@ en:
         answers: *YESNO
         absence_answer: ''
       is_overdrawn:
-        question: Is the account overdrawn?
+        question: Account overdrawn?
         answers: *YESNO
         absence_answer: ''
       provider_name:
-        question: What is the name of the bank, building society or other holder of the savings?
+        question: Bank or building society
         absence_answer: ''
       sort_code:
-        question: What is the sort code or branch name?
+        question: Sort code or branch name
         absence_answer: ''
       has_capital_savings:
-        question: Which savings does %{subject} have inside or outside the UK?
+        question: Savings %{subject} has 
         answers:
           'none': *absence_none
       # END saving
       #
       # START premium_bonds
       has_premium_bonds:
-        question: Does your client have any Premium Bonds?
+        question: Any Premium Bonds?
```

Comment:
Where is the source of truth for these now? I'm doing partner premium bonds and am seeing this Q as:
`Premium Bonds`

Reply:
@arthurashman I've seen both used, and a few inconsistencies, generally I've been using the RHS of figma: 
<img width="426" alt="Screenshot 2024-06-12 at 09 24 03" src="https://github.com/ministryofjustice/laa-apply-for-criminal-legal-aid/assets/34935/2f562e57-f8a4-4df0-b777-9186d98ae124">

Reply:
OK, ta